### PR TITLE
Fixup React errors

### PR DESCRIPTION
**Pull request by bronzehedwick**
Through code churn some React errors and warnings were introduced. Some
were just lint issues like unused variables, some improper accessibility
warnings, and some real errors around using `class` instead of
`className` inside JSX.

### CODE REVIEW

File: frontend/src/components/Mapbox.js [hunk 1]
```diff
@@ -58,35 +58,19 @@ const Mapbox = props => {
...
-          .addTo(map)
-          .togglePopup()
-        const normalMarker = new mapboxgl.Marker()
-          .setLngLat([props.voterData.longitude, props.voterData.latitude])
-          .setPopup(normalVotingPopup)
-          .addTo(map)
-          map.flyTo({
-            center:[props.voterData.ev_longitude,props.voterData.ev_latitude],zoom:13,
-            essential:true
-          })
```
Comment (ktomanelli) [inline frontend/src/components/Mapbox.js:82]: Looks like you're removing the map markers for the early and normal voting sites. I believe these are required as we have the popups attached to them. Not sure if the popups will be able to appear without the markers. The var names aren't in use though so if you're just trying to clean up un-used variables, maybe just remove the var names and leave the marker

**Review by ktomanelli — CHANGES_REQUESTED** [review]
Looks like you're removing the map markers from the

File: frontend/src/components/Mapbox.js (hunk 1 above)
Comment (bronzehedwick) [reply frontend/src/components/Mapbox.js:82]: Good catch, yeah I'll put this code back tonight, thanks.

**Comment by bronzehedwick** [comment]
@ktomanelli The code you mentioned has been put back, minus the `const` assignments, so the side-effects should still work.

**Review by ktomanelli — APPROVED** [review]
Awesome!

**Comment by azlyth** [comment]
LGTM

**Pull request merged**